migrations/2019-01-26-090348_create_version_owner_actions/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE version_owner_actions (
+    id SERIAL PRIMARY KEY,
+    version_id INTEGER REFERENCES versions(id) ON DELETE CASCADE,
```

Comment:
This is based on the previous commit, but my gut feeling is that version_id and owner_id should be NOT NULL. owner_token_id should possibly be also NOT NULL depending on other parts of this commit.

Reply:
This has been merged without NOT NULL constraints. Should we add them? Is it safe to add a NOT NULL constraint to a column?

Reply:
Whoops, yes, you're right-- these should be `NOT NULL` so that `VersionOwnerAction` doesn't need to have `Option` in `version_id`, `owner_id`, and `owner_token_id` columns. Nothing is adding data to this table right now, so it's safe to add `NOT NULL` constraints to these columns. Do you mind adding a migration doing that to this PR?

Reply:
Actually, thinking about it more, we don't currently allow publishing/yanking/unyanking from the UI (that is, session auth) but [there have been requests for at least yanking/unyanking](https://github.com/rust-lang/crates.io/issues/1659). So `owner_token_id` should NOT have a `NOT NULL` constraint, and it will be null if performed via the UI/session auth.